pkg/controller/container-runtime-config/helpers.go

```diff
@@ -462,7 +462,14 @@ func updatePolicyJSON(data []byte, internalBlocked, internalAllowed []string, re
 		}
 	}
 
-	policyObj.Transports["atomic"] = transportScopes
+	// three or more path segments are not supported by the atomic transport
+	policyObj.Transports["atomic"] = make(signature.PolicyTransportScopes)
+	for reg := range transportScopes {
```

Comment:
Non-blocking microoptimization:
`for scope, config := range transportScopes` … `policyObj.transports["atomic"][scope] = config` would avoid one hash lookup per loop